### ISSUE TITLE
Ormulum lineation

### ISSUE THREAD

**Issue by mmwwah** [scottkleinman/aeme#59]
We have talked around the question of lineation somewhat (particularly wrt caesuras), but I'm not sure we've addressed it directly.  

As you may know, H&W number (and typeset) by half-line, with the second half-line always indented.  I disapprove (and will address this in the notes/apparatus).  Will our front end have toggle for H&W line-numbering vs. Worley line-numbering?  Or should we do it the way that editions of Plato do it, giving both separated with a virgule?  Or can H&W just piss off?  Any thoughts?

This may or may not be a question to put to the advisory board and/or possible future users.  I am considering forming a mini-advisory board for the Ormulum (consisting of Adrienne, Carla, and possibly Mark Faulkner at Sheffield -- the three people who have looked at Junius 1 in the last decade), to whom I can pitch this sort of "What do users need?" question.

**Comment by scottkleinman** [comment]
I think there is some relevant discussion [here](https://github.com/scottkleinman/aeme/issues/24#issuecomment-92161279). The Laud Misc. 108 *Debate between the Body and Soul* is similar to the Ormulum, except that the AB rhyme on the half-line is a difference in syllable count. But you'll notice that the half-lines are rendered in `<p>` elements in html and the b-verse is brought inline with the a-verse using css. I think we can handle this sort of trick in the basic AEME browser. But you probably wouldn't want to follow the *Debate between the Body and Soul* exactly. If you did, the underlying markup would look like this:

```xml
<l xml:id="bodljunius1.1" rhyme="A">Nu, broþerr Wallterr, broþerr min</l>
<l  xml:id="bodljunius1.2" rhyme="B">Affterr þe flæshess kinde;</l>
```
In other words, you would be encoding H&W lineation. An alternative, is to use `@xml:id="bodljunius1.1a"` and `bodljunius1.1b`, etc. I'm not sure I like that. What about this?
```xml
<l xml:id="bodljunius1.1">
    <seg type="a-verse">Nu, broþerr Wallterr, broþerr min</seg>
    <seg type="b-verse">Affterr þe flæshess kinde;</seg>
</l>
```
(Of course, I'm leaving out punctuation issues here.) That would be really easy to render inline.

The disadvantage is that you would not be able to find lines in the H & W edition. I believe that PPEA had a way of dealing with line numbers in alternate editions. I'll go back and have a look before I make any suggestions.

**Comment by scottkleinman** [comment]
OK, the PPEA solution is to do something like this:

```xml
<l xml:id="bodljunius1.1" n="HW.1">Nu, broþerr Wallterr, broþerr min</l>
<l xml:id="bodljunius1.2" n="HW.2">Affterr þe flæshess kinde;</l>
```
But this ignores the half-line issue. So maybe something like this:

```xml
<l xml:id="bodljunius1.1">
    <seg type="a-verse" n="HW.1">Nu, broþerr Wallterr, broþerr min</seg>
    <seg type="b-verse" n="HW.2">Affterr þe flæshess kinde;</seg>
</l>
```
We could possibly automate this, but one irregularity would require adjustments.

**Comment by mmwwah** [comment]
Hmm.  I'm not sure what to do, or what to advocate for.  I'm going to leave this open for now, and we can grapple with it later, if that's okay.  

Partly I'm blinded by the fact that I don't do anything with the Ormulum that requires attention to half-lines, so the civilian/lay side of me is all, like, "Oh who cares about that?"  Consultation with kitchen cabinet, as well as further work encoding, should improve my vision.

**Comment by mmwwah** [comment]
Okay, I've changed my mind.  Until NPJohannesson's EETS edition comes out (and he's a dozen years behind schedule, according to Vincent Wimbush), people will be navigating by H&W numbering.  I think I'm going to go with that after all.

Any reason I shouldn't, or other things to consider?

**Comment by scottkleinman** [comment]
It might be a good idea to use the `@n` technique in case you end up with a different number of lines. If your lineation ends up exactly the same as theirs, you can delete the extra markup later. Alternatively, you can check for discrepancies at the end of the editing process and add `@n` throughout if it proves necessary. Either way, it will unfortunately involve some tedious work.

**Comment by dorothyk98** [comment]
I know we've talked about this separately Meg, but tell me if you want to do a conference Facebook or other forum talk with Adrienne and Carla to figure out how to deal with this. Clearly we are going to have to come up with the correct foliation (the numbers are a mess) and this also deals with the lineation issue.
